docs/cookbook/getSortedLocalDateTimes.mjs

```diff
@@ -0,0 +1,24 @@
+import assert from "assert";
+/**
+ * getSortedLocalDateTimes will sort an array of zoneless Temporal.DateTime instances by the
+ * corresponding local date and time of day (e.g., for building a conference schedule).
+ *
+ *
+ * @param {Temporal.DateTime[]} dateTimes This is a Date instance
```

Comment:
The documentation of the parameters needs a bit of cleaning up.
- "Date instance" → "DateTime instance"
- "direction" → "reverse"
- "Temporal.Absolute instance" → "the array from dateTimes, sorted"(?)